### PR TITLE
Hotfix/2020.1.4

### PR DESCRIPTION
**Pull request by PenghaiZhang**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [ ] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

This Hotfix will fix the bug about failing to upgrade from 2019.2.4 to 2020.1.
I have tested upgrading from 2019.1.6 and 2019.2.4 and both succeeded.

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/plugin-jpf.xml [hunk 1]
```diff
@@ -5148,6 +5148,13 @@
     <parameter id="bean" value="bean:com.tle.core.institution.migration.v20192.AddLastOwnerMigration" />
     <parameter id="date" value="2019-09-17" />
   </extension>
+  <!-- Let 2020.1 know v20192.RemoveLastKnownUserConstraint-->
+  <extension plugin-id="com.tle.core.migration" point-id="migration" id="RemoveLastKnownUserConstraint20192">
+    <parameter id="id" value="com.tle.core.institution.migration.v20192.RemoveLastKnownUserConstraint" />
+    <parameter id="bean" value="bean:com.tle.core.institution.migration.v20192.RemoveLastKnownUserConstraint" />
+    <parameter id="date" value="2020-08-31" />
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/plugin-jpf.xml:5155]: As mentioned in the other one, maybe tweak this date. ;)

**Review by edalex-ian — APPROVED** [review]
LGTM! :+1:

**Comment by PenghaiZhang** [comment]
Manually done testing includes:
1. from 2020.1.3 to 2020.1.4;
2. from 2019.2.4 to 2020.1.4;
3. from 2019.1.6 to 2020.1.4;
4. from 2020.1.4 to the latest.

**Review by edalex-ian — APPROVED** [review]

**Comment by SammyIsConfused** [comment]
Tested. Looks good to me.

**Pull request merged**